src/worker.js

```diff
@@ -52,16 +52,33 @@ function processTile(tile, callback) {
     }
 
     var writeQueue = queue(1);
+    var writeBuffer = [];
+
+    function prepWriteBuffer() {
+      return writeBuffer.splice(0).map(function(ele) {
+        return (typeof ele !== 'string' ? JSON.stringify(ele) : ele) + '\x1e';
+      }).join('');
```

Comment:
I think it would be both faster and simpler to have a string `writeBuffer` instead of an array, and stringify immediately instead of push. Less copy, less memory, less GC.